tools/migration/internal/repo_fs_wrangler.go

```diff
@@ -37,13 +40,33 @@ func (rmh *RepoFSWrangler) GetNewRepoPath() string {
 	return rmh.newRepoPath
 }
 
-// MakeNewRepo creates the new repo dir and returns it with Read/Write access.
-//   This returns an error if the directory exists.
-func (rmh *RepoFSWrangler) MakeNewRepo() (*os.File, error) {
-	if err := os.Mkdir(rmh.newRepoPath, os.ModeDir|0744); err != nil {
-		return nil, err
+// MakeNewRepo copies the old repo to the new repo dir with Read/Write access.
+//   Returns an error if the directory exists.
+func (rmh *RepoFSWrangler) MakeNewRepo() error {
```

Comment:
Rename this to more clearly imply the copy operation, e.g. `CloneRepo`. Return the path to the clone. As I noted above, I think this should find a free directory name rather than fail, but if not then take the intended migration version number here rather than at construction.

Reply:
+1 `CloneRepo` is clearer
+1 version number seems unimportant in name of the copy
+1 that if we do want the version number here the wrangler can read it within this subfunction, probably by calling a `RepoVersion` method on the wrangler